x-pack/plugin/src/test/resources/rest-api-spec/test/flattened/20_flattened_stats.yml

```diff
@@ -0,0 +1,41 @@
+setup:
+  - skip:
+      version: " - 7.99.99"
```

Comment:
@jtibshirani Should this one be `7.5.99` now?

Reply:
Oops, thanks @jpountz, I missed this when updating the serialization version.